website/pages/docs/job-specification/group.mdx

```diff
@@ -129,12 +141,35 @@ group "example" {
 }
 ```
 
+### Stop on Client Disconnect
+
+This example shows how `stop_on_client_disconnect` interacts with
+other group stanzas. After the default 10 second [`heartbeat_grace`]
+window expires, the server will reschedule both workloads. The client
+will wait 90 seconds before sending a stop signal (`SIGTERM`) to the
+`first` group. After 15 more seconds because of the `shutdown_delay`,
+the client will send `SIGKILL`. The `second` group does not have
+`stop_on_client_disconnect` and will not be stopped on the client no
+matter how long the client is out of touch.
```

Comment:
```suggestion
`stop_on_client_disconnect` and will not be stopped on the client, regardless
of how long the client is unable to contact a server.
```

Reply:
Done. (Had to reflow so I just picked that up in its own commit)